internal/config/config.go

```diff
@@ -231,7 +231,11 @@ func Load(path string) error {
 	// Set default registry auth config path
 	viper.SetDefault(RegistryCredentialConfigKey, DefaultRegistryCredentialConfPath)
 	// Set default driver
-	viper.SetDefault(DriverKey, DefaultDriver)
+	viper.SetDefault(DriverTypeKey, DefaultDriver.Type)
```

Comment:
This fixes the retrieving of `DriverFooKey` when falcoctl.yaml is an empty file (as used in tests).
Otherwise `Driverer` would find an empty `DriverTypeKey` value and fail during the conversion to a `drivertype.Type`.